pkg/cmd/attestation/verify/verify.go

```diff
@@ -84,6 +84,20 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 			// set policy flags based on what has been provided
 			opts.SetPolicyFlags()
 
+			// check if we will be calling GitHub APIs
+			if opts.BundlePath == "" {
+				// If so, ensure user is authenticated
+				factoryConfig, err := f.Config()
+				if err != nil {
+					return err
+				}
+
+				authOk := cmdutil.CheckAuth(factoryConfig)
+				if !authOk {
+					return fmt.Errorf("Please supply a bundle or authenticate to gh")
```

Comment:
I though about using `pkg/cmd/root/help.go:authHelp()` here but, (1) importing `root` here would create a circular dependency and (2) it's a private function. I'm open to feedback here!

Reply:
Realistically if we go down this route I'd imagine that we have something like `cmderrs` package used like `cmderrs.AuthRequired` or something like that.